src/containers/AboutPage/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Row, Col } from "antd";
+import kid1Img from "./../../images/aboutPage/kid-1.png";
+import kid2Img from "./../../images/aboutPage/kid-2.png";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+
+const AboutPage = () => {
+  const [t] = useTranslation();
+
+  return (
+    <div className="aboutUs">
+      <Row className="aboutUs__upperContent">
+        <Col className="aboutUs__textCol">
+          <Row className="aboutUs__textCol--aboutText">
```

Comment:
For the styling of About us paragraph, you may add `text-align: justify;` it's a suggestion because I'm not sure how this styling will look in Arabic version. You may consider it also for the Why do we do it paragraph.

Reply:
I used the `text-align` as suggested, and I used the the "Salsa" font for the headers but kept the paragraph's text as default because it's important to be readable, as suggested by Berfin. regarding the colors and the size I didn't modify them because as we agreed in the last standup meeting we need to discuss those and agree on one color for the context's text and their readability. 

PTAL

Reply:
You are exactly right, we should decide and go with using one of them.